--- archive.h	2020-09-11 14:23:21.758842500 +0000
+++ archive2.h	2020-09-11 14:20:27.310494460 +0000
@@ -840,11 +840,11 @@
  */
 __LA_DECL int archive_write_header(struct archive *,
 		     struct archive_entry *);
-__LA_DECL la_ssize_t	archive_write_data(struct archive *,
+__LA_DECL int	archive_write_data(struct archive *,
 			    const void *, size_t);
 
 /* This interface is currently only available for archive_write_disk handles.  */
-__LA_DECL la_ssize_t	 archive_write_data_block(struct archive *,
+__LA_DECL int	 archive_write_data_block(struct archive *,
 				    const void *, size_t, la_int64_t);
 
 __LA_DECL int		 archive_write_finish_entry(struct archive *);
