--- archive_virtual.c	2020-09-11 16:39:07.158014139 +0000
+++ archive_virtual2.c	2020-09-11 16:39:50.842107856 +0000
@@ -124,13 +124,13 @@
 	return ((a->vtable->archive_write_finish_entry)(a));
 }
 
-la_ssize_t
+int
 archive_write_data(struct archive *a, const void *buff, size_t s)
 {
 	return ((a->vtable->archive_write_data)(a, buff, s));
 }
 
-la_ssize_t
+int
 archive_write_data_block(struct archive *a, const void *buff, size_t s,
     la_int64_t o)
 {
