--- CMakeLists.txt
+++ CMakeLists.txt
@@ -59,6 +59,7 @@ include(GNUInstallDirs)
 # Configuration
 #
 option(BUILD_SHARED_LIBS "Build shared libraries (rather than static ones)" ON)
+option(HIDE_SYMBOLS "Build a libraries with hidden symbols unless they're specifically exported" ON)
 option(URIPARSER_BUILD_DOCS "Build API documentation (requires Doxygen, Graphviz, and (optional) Qt's qhelpgenerator)" ON)
 option(URIPARSER_BUILD_TESTS "Build test suite (requires GTest >=1.8.0)" ON)
 option(URIPARSER_BUILD_TOOLS "Build tools (e.g. CLI \"uriparse\")" ON)
@@ -104,10 +105,12 @@ endmacro()
 #
 set(URIPARSER_EXTRA_COMPILE_FLAGS)
 
+if(HIDE_SYMBOLS)
 check_c_compiler_flag("-fvisibility=hidden" URIPARSER_COMPILER_SUPPORTS_VISIBILITY)
 if(URIPARSER_COMPILER_SUPPORTS_VISIBILITY)
     set(URIPARSER_EXTRA_COMPILE_FLAGS "${URIPARSER_EXTRA_COMPILE_FLAGS} -fvisibility=hidden")
 endif()
+endif(HIDE_SYMBOLS)
 
 #
 # config.h
